modules/test/playwright/pages/document-library-web/gogoShell.page.ts

```diff
@@ -0,0 +1,36 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2000 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+// @ts-ignore
```

Comment:
why this `// @ts-ignore`?

Reply:
Removed for every page in `document-library-web`